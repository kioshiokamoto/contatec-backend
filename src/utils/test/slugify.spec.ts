import { slugify } from '../slugify';
describe('Utilidad de funcion para generar slugs', () => {
  it('Funcion debe poder ser llamada', () => {
    const slug = slugify('test diego');
    expect(slugify).toMatchSnapshot();
  });
  it('Funcion debe retornar slug correcatamente', () => {
    const slug = slugify('test diego');
    expect(slug).toBe('test_diego');
  });
});
