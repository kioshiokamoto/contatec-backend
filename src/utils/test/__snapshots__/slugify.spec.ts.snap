// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utilidad de funcion para generar slugs Funcion debe poder ser llamada 1`] = `[Function]`;
